studio/lib/constants/index.ts

```diff
@@ -2,6 +2,7 @@ export * from './infrastructure'
 export * from './metrics'
 
 export const IS_PLATFORM = process.env.NEXT_PUBLIC_IS_PLATFORM === 'true'
+export const DEFAULT_HOME = IS_PLATFORM ? '/projects' : 'project/default'
```

Comment:
Why does `/projects` have a leading `/`, but `project/default` does not?

Reply:
oh good catch! 🙏